code/game/gamemodes/cult/runes/rune.dm

```diff
@@ -21,13 +21,23 @@
 	QDEL_NULL(rune)
 	return ..()
 
-/obj/effect/rune/examine(mob/user)
-	..(user)
+/obj/effect/rune/examine(mob/user, var/distance = -1)
+	
+	//since runes are drawn with magic blood-infused chalk now, we are overriding the examine proc so they aren't visibly blood-stained, even though they have blood in them
+	to_chat(user, desc)
```

Comment:
Can you still examine this if you're unconscious?

Reply:
I tested; no, you can't.  But the way I did it did kinda mess it up.  It should be fixed now, I think.

Reply:
I'll mark this as resolved?  I'm not sure how this works.